pipelines/zap/pipeline.yml

```diff
@@ -43,69 +48,35 @@ resources:
     options:
       - "--exclude '*'"
       - "--include 'results/*'"
-- name: s3-bucket-summary
-  type: s3-resource
-  source:
-    access_key_id: {{aws-access-key}}
-    secret_access_key: {{aws-secret-key}}
-    bucket: {{aws-bucket}}
-    options:
-      - "--exclude '*'"
-      - "--include 'zap-summary/summary.json'"
-- name: fetch-project-data
+<% projects.each do |project| -%>
+- name: fetch-project-data-<%= project['name'] %>
   type: http-resource
   source:
-    uri: https://team-api.18f.gov/public/api/projects/
-    filename: projects.json
-    index: https://team-api.18f.gov/public/api/projects/
+    uri: https://team-api.18f.gov/public/api/projects/<%= project['name'].downcase %>/
+    filename: project.json
+    index: https://team-api.18f.gov/public/api/projects/<%= project['name'].downcase %>/
     etag: true
+<% end -%>
 
 jobs:
-- name: zap-scheduled
+<% projects.each do |project| -%>
+<% %w(scheduled ondemand).each do |job| -%>
+- name: zap-<%= job %>-<%= project['name'] %>
+<% if job == 'scheduled' -%>
   # use of the `time` Resource blocks manual triggering, so might as well remove the option
   disable_manual_trigger: true
+<% end -%>
   plan:
+<% if job == 'scheduled' -%>
   - get: after-midnight
     trigger: true
+<% end -%>
   - get: scripts
-  - get: projects-json
-    resource: fetch-project-data
-  - get: s3-bucket-results
-  - task: filter-team-data
-    file: scripts/tasks/filter-team-data/task.yml
-  - task: fetch-last-zap-results
-    file: scripts/tasks/fetch-last-zap-results.yml
-  - task: scan-zap
-    file: scripts/tasks/run-zap/task.yml
-  - put: s3-bucket-results
-  - task: summarize-zap-results
-    file: scripts/tasks/summarize-zap-results/task.yml
-  - put: s3-bucket-summary
-    on_success:
-      put: slack
-      params:
-        channel: {{slack-channel}}
-        icon_url: {{slack-icon}}
-        text_file: zap-summary/summary.txt
-        username: {{slack-user}}
-    on_failure:
-      put: slack
-      params:
-        text: |
-          :x: FAILED to scan properties.
-          <https://ci.cloud.gov/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
-        channel: {{slack-channel}}
-        username: {{slack-user}}
-        icon_url: {{slack-icon}}
-
-- name: zap-ondemand
-  plan:
-  - get: scripts
-  - get: projects-json
-    resource: fetch-project-data
+  - get: project-data
+    resource: fetch-project-data-<%= project['name'] %>
   - get: s3-bucket-results
-  - task: filter-team-data
-    file: scripts/tasks/filter-team-data/task.yml
+  - task: filter-project-data
+    file: scripts/tasks/filter-project-data/task.yml
   - task: fetch-last-zap-results
     file: scripts/tasks/fetch-last-zap-results.yml
   - task: scan-zap
```

Comment:
It'd be nice to have some sort of notification that the pipeline failed, but we certainly don't want to be notified for *every single job*.